tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -27,13 +27,36 @@ parameters:
   type: string
   default:
 
+- name: isClientUtilsPackage
+  type: boolean
+  default: false
+
 steps:
 - task: Bash@3
   displayName: Generate .npmrc for ${{ parameters.artifactPath }}
   inputs:
     targetType: 'inline'
     workingDirectory: $(Pipeline.Workspace)/pack/${{ parameters.artifactPath }}
-    ${{ if eq(parameters.namespace, true) }}:
+    ${{ if eq(parameters.isClientUtilsPackage, true) }}:
```

Comment:
All this bash scripting makes me very nervous. Also, why do we need such a specific parameter? Why do we have to special case this one package?

Reply:
Because we only want to release this internal package to a build feed and rest internal packages(which will be renamed to fluid-private) to dev feed. In npmrc we can add different registries with scope, however currently they have same scope i.e fluid-internal. So I added a separate step to publish this to build feed. 

This work is temporary. Once the rename happens, we can remove all this.

Reply:
I think a better approach would be to create a CLI command that, when given a package name, outputs the publishing policy that should apply to that package. We already have most of that logic because we check scopes and private/not private in policy check already. Basically we need to move towards a design where this logic isn't basked in/hardcoded in CI.

Reply:
That is good for long term approach but as we have tested it, so this seems ok for a temporary change so that we don't have to manually release this package again.